### ISSUE TITLE
the pointer interaction breaks on mobile when you drag vertically

### ISSUE THREAD

**Issue by Fil** [observablehq/plot#2157]
On mobile, pointing works if you tap on a point, and it also works if you touch and drag horizontally. But as soon as you drag 1 pixel vertically (or sideways), the page moves up/down, and the interaction stops. 

Not sure if this is a 100% correct description. Tested in an observable notebook (I'd want to try and replicate it on a framework page).

**Comment by mbostock** [comment]
Not sure we want to change this; doing so would presumably prevent vertical scrolling if that is what you intended by touching and dragging and simply happened to touch the chart to start. We don’t want to create “dead areas” where you can’t scroll because the chart is stealing the input.

**Comment by Fil** [comment]
Yes that's an important design constraint. But it still feels slightly broken as is, when I compare against [this smooth chart](https://www.nytimes.com/interactive/2024/us/elections/polls-president.html). In the NYT chart dragging does not break the interaction,  and at the same time does not create a dead zone.
